docker/Dockerfile

```diff
@@ -35,9 +39,9 @@ ENV PATH=/home/work/incubator-superset/superset/bin:$PATH \
     PYTHONPATH=./superset/:$PYTHONPATH
 
 COPY docker-init.sh /usr/local/bin/
-RUN chmod +x /usr/local/bin/docker-init.sh
+
+RUN chmod +x /usr/local/bin
```

Comment:
Why this change. I see no harm but what trigger this as only file we copy to /usr/local/bin is docker-init.sh

Reply:
this change was required as to give permission to bin directory as shell script used to trigger the kerberos ticket validation resides under `bin` directory and since its dynamically created via config map so it will be available later so couldn't give permission to the file here